uisdk/src/braintree/res/values/colors.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Braintree -->
+    <!-- New Colors -->
+    <color name="kh_uisdk_primary">#18183F</color>
```

Comment:
Will it work if DP will override 'kh_uisdk_primary', 'kh_uisdk_secondary' or 'kh_uisdk_accent'?

Reply:
Yes, it will be overriden